### PR TITLE
Raven msg

### PR DESCRIPTION
**Pull request by ryandezinator**

### CODE REVIEW

File: interpretation/ww_head.py [hunk 1]
```diff
@@ -677,19 +677,23 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
...
             if is_command(message, ['end', 'prevent', 'reveal', 'stop']) and user_role == "Royal Knight":
-                # TODO
+                # TODO             
                 return todo()
             if is_command(message, ['end', 'prevent', 'reveal', 'stop'], True) and user_role == "Royal Knight":
-                # TODO
-                return todo()
+                msg = "**Usage:** Prevent a lynch.\n\n`" + prefix + "prevent <player>`\n\n"
+                msg += "**Example:** `" + prefix + "prevent @Randium#6521`\nThe command is compatible with emojis as a replacement for user mentions. "
+                msg += "This command can only be used by the Royal Knight."
```
Comment (Randium) [inline interpretation/ww_head.py]: LOOOOOOL you can't add a target to your lynch prevention. 😂 Either you prevent the lynch or you don't, there's no addition of a user!

File: interpretation/ww_head.py [hunk 2]
```diff
@@ -699,8 +703,10 @@ def process(message, isGameMaster=False, isAdmin=False, isPeasant=False):
                 # TODO
                 return todo()
             if is_command(message, ['heal', 'life', 'save'], True) and user_role == "Witch":
-                # TODO
-                return todo()
+                msg = "**Usage:** Use potion of life.\n\n`" + prefix + "save <player>`\n\n"
+                msg += "**Example:** `" + prefix + "save @Randium#6521`\nThe command is compatible with emojis as a replacement for user mentions. "
+                msg += "This command can only be used by the Witch."
```
Comment (Randium) [inline interpretation/ww_head.py]: Same for this one. Your potion will save EVERYBODY that is getting attacked during the night. No mentioning players.

File: interpretation/ww_head.py (hunk 1 above)
Comment (ryandezinator) [reply interpretation/ww_head.py]: oh XD

File: interpretation/ww_head.py (hunk 2 above)
Comment (ryandezinator) [reply interpretation/ww_head.py]: LMAO

**Pull request merged**